README.md

```diff
@@ -57,7 +54,7 @@ Note: If installing via pip fails due to pip failing to upgrade your system Pyth
 
 #### Via curl
 
-Note: You may need to adjust the target desination to match a directory in your `$PATH`.
+Note: You may need to adjust the target destination to match a directory in your `$PATH`.
```

Comment:
This was the only real change I made.... everything else is just prettier 🙈

Reply:
Hehe fanks for the proofreading :P